getting-started/CONCEPTS/ACTION-PLANNER.md

```diff
@@ -0,0 +1,25 @@
+# Action Planner
+
+The Action Planner is a powerful planner that uses a LLM to generate plans. It can trigger parameterized actions and send text based responses to the user. It supports the following advanced features:
```

Comment:
```suggestion
The Action Planner is a powerful planner that uses an LLM to generate plans. It can trigger parameterized actions and send text-based responses to the user. It supports the following advanced features:
```